plugins/doc_fragments/ipamodule_base_docs.py

```diff
@@ -30,4 +30,8 @@ class ModuleDocFragment(object):  # pylint: disable=R0205,R0903
   ipaadmin_password:
     description: The admin password.
     required: false
+  ipaapi_ldap_cache:
+    description: Set IPA connecton ldap_cache.
```

Comment:
Please enhance the description here also.

Reply:
Fixed.